lib/rubocop/cop/mixin/target_ruby_version.rb

```diff
@@ -5,15 +5,24 @@ module Cop
     # Common functionality for checking target ruby version.
     module TargetRubyVersion
       def required_minimum_ruby_version
-        @minimum_target_ruby_version
+        @minimum_target_ruby_version || 0
```

Comment:
Can you check the documentation task? I think this will cause it to display a wonky required ruby version for this cop.

Reply:
Oh good call, you're probably right.

Reply:
Fixed now.